site/cds_rdm/legacy/redirector.py

```diff
@@ -56,11 +58,23 @@ def legacy_files_redirect(legacy_id, filename):
     version = query_params.pop("version", None)
     try:
         record = get_record_by_version(parent_pid.pid_value, version)
```

Comment:
What happens here if `version` is not passed in the params?

Reply:
`if not version` takes care of that, this was already there :D